conference.js

```diff
@@ -303,6 +305,15 @@ class ConferenceConnector {
             this._handleConferenceJoined.bind(this));
         room.on(JitsiConferenceEvents.CONFERENCE_FAILED,
             this._onConferenceFailed.bind(this));
+        room.on(JitsiConferenceEvents.TRACK_OWNER_CHANGED, (ssrc, owner) => {
```

Comment:
We need to add an event listener [here](https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/conference/actions.js#L96) as well for ssrc-rewriting to work on react-native clients.

Reply:
done